crates/sui-types/src/temporary_store.rs

```diff
@@ -293,64 +266,6 @@ impl<S> TemporaryStore<S> {
         }
     }
 
-    /// Compute storage gas for each mutable input object (including the gas coin), and each created object.
-    /// Compute storage refunds for each deleted object
-    /// Will *not* charge any computation gas. Returns the total size in bytes of all deleted objects + all mutated objects,
-    /// which the caller can use to charge computation gas
-    fn charge_gas_for_storage_changes(
```

Comment:
nit, moving these functions makes it a bit hard to review

Reply:
I know :/, but it has to be done because these functions now require `S: ObjectStore`